lasagne/layers/conv.py

```diff
@@ -80,6 +82,74 @@ def conv_output_length(input_length, filter_size, stride, pad=0):
     return output_length
 
 
+def conv_input_length(output_length, filter_size, stride, pad=0):
+    """Helper function to compute the input size of a convolution operation
```

Comment:
But then we'd need all the explanations for the stride/dilation and padding/cropping we have in `TransposedConv2DLayer`. I find it much easier to document this as computing the input size of a convolution for which you know the output size.